transforms/class.js

```diff
@@ -1103,23 +1118,80 @@ module.exports = (file, api, options) => {
       return false;
     };
 
+    const reinsertTopComments = () => {
+      root.get().node.comments = topComments;
+    };
+
     // the only time that we can't simply replace the createClass call path
     // with a new class is when the parent of that is a variable declaration.
     // let's delay it and figure it out later (by looking at `path.parentPath`)
     // in `updateToClass`.
-    const apply = (path) =>
-      path
-        .filter(mixinsFilter)
-        .filter(hasNoCallsToDeprecatedAPIs)
-        .filter(hasNoRefsToAPIsThatWillBeRemoved)
-        .filter(doesNotUseArguments)
-        .filter(isInitialStateConvertible)
-        .filter(canConvertToClass)
-        .forEach(updateToClass);
-
-    const didTransform = apply(
-      ReactUtils.findAllReactCreateClassCalls(root)
-    ).size() > 0;
+    const apply = (path) => {
+      const result = {};
+      path.forEach(childPath => {
+        if (
+          mixinsFilter(childPath) &&
+          hasNoCallsToDeprecatedAPIs(childPath) &&
+          hasNoRefsToAPIsThatWillBeRemoved(childPath) &&
+          doesNotUseArguments(childPath) &&
+          isInitialStateConvertible(childPath) &&
+          canConvertToClass(childPath)
+        ) {
+          result.didTransform = true;
+          updateToClass(childPath);
+        } else {
+          result.didFallback = true;
+          fallbackToCreateClassModule(childPath);
+        }
+      });
+      return result;
+    };
+
+    const {
+      didTransform,
+      didFallback,
+    } = apply(ReactUtils.findAllReactCreateClassCalls(root));
+
+    if (didFallback) {
+      const reactPathAndBinding =
+        findRequirePathAndBinding('react') ||
+        findRequirePathAndBinding('React');
+
+      if (reactPathAndBinding) {
+        const {path, type} = reactPathAndBinding;
+        const noReactReferences = root
+          .find(j.Identifier)
+          .filter(path => path.value.name === 'React')
+          .length === 1;
+        const noJSX = root.find(j.JSXElement).length === 0;
+        const shouldRemoveReactImport = noReactReferences && noJSX;
```

Comment:
Oh, sorry I missed this part. If you run it on https://github.com/airbnb/enzyme/blob/master/src/ReactWrapperComponent.jsx you'll see the `PropTypes` import also gets removed.

This is only an issue when it's a `import` statement and has multiple specifiers on the left hand side. Let's add another guard for it?